compress/src/lib.rs

```diff
@@ -0,0 +1,26 @@
+/// Compresses the passed string into the output byte vector.
+pub fn compress(input: String) -> Vec<u8> {
+    let input = input.into_bytes();
+    let mut output = Vec::with_capacity(input.len());
+    brotli::BrotliCompress(&mut input.as_slice(), &mut output, &Default::default()).unwrap();
```

Comment:
Instead of the default parameters we could probably specify a [`quality`](https://docs.rs/brotli/latest/brotli/enc/backward_references/struct.BrotliEncoderParams.html#structfield.quality) to get even smaller things.